README.md

```diff
@@ -212,8 +212,13 @@ ESLint doesn't natively support experimental ECMAScript language features. You c
 
 Once a language feature has been adopted into the ECMAScript standard, we will accept
 issues and pull requests related to the new feature, subject to our [contributing
-guidelines](http://eslint.org/docs/developer-guide/contributing). Until then, please use
-the appropriate parser and plugin(s) for your experimental feature.
+guidelines](http://eslint.org/docs/developer-guide/contributing). Note that we will
```

Comment:
Fair enough, I'll split this into two PRs. This one will just add the parenthetical and can hopefully be merged with minimal discussion; and the other will add another FAQ for why not stage 3.

My original motivation was to answer the stage 3 question, which does come up sometimes in chat. The goal being that someone can just c/p a deep link to that question and call it a day, instead of repeatedly justifying why not stage 3. But I'm certainly okay with working our way there over a couple of PRs.